src/components/panel/panel.js

```diff
@@ -284,6 +284,15 @@ angular
  * toggled the class and animations are completed.
  */
 
+/**
+ * @ngdoc method
+ * @name MdPanelRef#focusOnOpen
+ * @description
+ * Focuses the panel content if the focusOnOpen config value is true. This should
```

Comment:
I agree, but the problem is do we call it on addClass or removeClass? If you have a class that is "display:none" you would need it on the remove class. I also can't check the visibility/display/etc of the panel since I don't know how the user will be hiding it.

Do you have a solution?